tests/dragonfly/replication_test.py

```diff
@@ -3173,13 +3173,21 @@ async def test_partial_replication_on_same_source_master_with_replica_lsn_inc(df
     await c_s2.execute_command(f"REPLICAOF NO ONE")
     # Make server 4 replica of server 2
     await c_s4.execute_command(f"REPLICAOF localhost {server2.port}")
+    await check_all_replicas_finished([c_s4], c_s2)
```

Comment:
I moved it here because the for statement bellow will seed a few values that won't reach the journal (because it's kinda racy if the journal gets registered by full sync before the first element seeded reaches dragonfly). To avoid this and force all the seeded values below to go through the journal, I moved the statement above. This does not affect the test